clang/test/dpct/cmake_migation/case_013/expected.txt

```diff
@@ -0,0 +1,12 @@
+set(cufile file1.cu)
```

Comment:
Line1 is not an correct expected output, should be "set(cufile file1.dp.cpp)" instead.

Reply:
The expected.txt file is now updated and the test now runs as expected

./ninja_release/./bin/llvm-lit -vv SYCLomatic/clang/test/dpct/cmake_migation/case_013/case_013_cuda_compile_cubin.cpp 
llvm-lit: workspace/JIRA/5673_CMake_Mig_Lit/cubin/SYCLomatic/llvm/utils/lit/lit/llvm/config.py:488: note: using clang: workspace/JIRA/5673_CMake_Mig_Lit/cubin/ninja_release/bin/clang
-- Testing: 1 tests, 1 workers --
PASS: Clang :: dpct/cmake_migation/case_013/case_013_cuda_compile_cubin.cpp (1 of 1)

Testing Time: 0.14s

Total Discovered Tests: 1
  Passed: 1 (100.00%)